src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -28,7 +28,7 @@ class grpc_ssl_credentials : public grpc_channel_credentials {
  public:
   grpc_ssl_credentials(const char* pem_root_certs,
                        grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
-                       const verify_peer_options* verify_options);
+                       const grpc_verify_peer_options* verify_options);
```

Comment:
The signature has changed, is this okay with our versioning policy since it introduces binary incompatibility?

Reply:
Could you please be more specific about "versioning policy" and "binary incompatibility"? 

IIUC, there is no requirement to retain backward-compatibility in C-core.

Reply:
If you have client code compiled using previous `grpc_ssl_credentials` class definition, the signature of the constructor `grpc_ssl_credentials` doesn't match the new signature. Wouldn't you get build/run-time failures because of this mismatch?

Reply:
It could break client code if client code is using the old field. No code in google3 or in grpc-github use this field. nullptr is passed. 

Since we do not need to maintain backward-compatibility, we just replace this API and make sure we do not break ourselves.

Reply:
Since "we do not need to maintain backward-compatibility", I have no issues with this change

Reply:
Note: when c-core backward compatibility is broken, we bump up the core version in the build files.

Reply:
@srini100 

Srini, could you please provide more details on how to bump the version number? e.g., what value should it bump to? I found core version appearing in both BUILD and build.yaml files.

Reply:
Bump core_version in both files to 8.0.0.